tests/test_entity_utils.py

```
@@ -9,15 +11,17 @@ def setUpClass(cls):
         modules = {
             'torch': MagicMock(),
             'transformers': MagicMock(),
-            'vector_database': MagicMock(),
+            'opendiscourse.vector_database': MagicMock(),
             'psycopg2': MagicMock(),
             'psycopg2.extras': MagicMock(),
             'dotenv': MagicMock(),
             'numpy': MagicMock(),
         }
         with patch.dict('sys.modules', modules):
+            project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```

Ruff (0.11.9)

Line too long (89 > 88)

(E501)